draft-ietf-v6ops-464xlaton.xml

```diff
@@ -383,7 +399,7 @@ NEW TEXT:
 <t>
 ===
 </t>
-<t>464XLAT-4: The IPv6 Transition CE Router MUST implement <xref target="RFC8781"/> ("Discovering PREF64 in Router Advertisements") and <xref target="RFC7050"/>
+<t>464XLAT-4: The IPv6 Transition CE Router MUST implement <xref target="RFC8781"/> ("Discovering PREF64 in Router Advertisements") and SHOULD implement <xref target="RFC7050"/>
```

Comment:
let's talk -- since we are trying to deprecate 7050, I say we take it straight down to a MAY and ack Nick's draft as an Informational reference.